boards/xtensa/xiao_esp32s3/xiao_esp32s3.dts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2023 Seeed Studio inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32s3.dtsi>
+#include "xiao_esp32s3-pinctrl.dtsi"
+#include "seeed_xiao_connector.dtsi"
+
+/ {
+	model = "Seeed XIAO ESP32S3";
+	compatible = "seeed,xiao-esp32s3";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &usb_serial;
+		zephyr,shell-uart = &usb_serial;
+		zephyr,flash = &flash0;
+	};
+
+	aliases {
+		i2c-0 = &i2c0;
+		watchdog0 = &wdt0;
+		led0 = &led0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 21 GPIO_ACTIVE_LOW>;
+			label = "BUILTIN LED";
+		};
+	};
+
+	power-states {
```

Comment:
I think that ESP32-S3 has no low power implemented yet. This can be removed.

Reply:
It is WIP: https://github.com/zephyrproject-rtos/hal_espressif/pull/210

Reply:
ok, I removed it now.